x-pack/plugins/security_solution/public/common/components/ml_popover/ml_modules.tsx

```diff
@@ -21,4 +21,6 @@ export const mlModules: string[] = [
   'security_linux',
   'security_network',
   'security_windows',
+  'security_linux_v3', 
```

Comment:
```suggestion
  'security_linux_v3',
```
The suggested change above (remove trailing space from line) should fix the linter error.

Reply:
Yes I was helped previously, just fixed that, thank you!